src/data/api-service.ts

```diff
@@ -13,7 +13,8 @@ const DEFAULT_REQUEST_TIMEOUT = 30000
  */
 export const getResourceSlug = (resource: string): string => {
   const slug = { [AUDIO]: 'audio', [IMAGE]: 'images' }[resource] ?? resource
-  return `${slug}/`
+  const noTrailingSlash = ['rate_limit'] // TODO: Ensure all API endpoints have trailing slashes
```

Comment:
Is this something that needs to be done in the API? If so, maybe we can add a link to the API issue here to follow up?

Reply:
It's already fixed in the API: https://github.com/WordPress/openverse-api/pull/801

This condition needs to stay, however, as the API isn't deployed with that fix yet.

Reply:
@sarayourfriend , I think the API was deployed yesterday?

Reply:
Yup! Thanks :+1: